airbyte-integrations/connectors/source-s3/setup.py

```diff
@@ -6,15 +6,15 @@
 from setuptools import find_packages, setup
 
 MAIN_REQUIREMENTS = [
-    "airbyte-cdk[file-based]>=0.57.7",
+    "airbyte-cdk[file-based]==0.59.2",  # temporarily pin until concurrency can be released
     "smart-open[s3]==5.1.0",
     "wcmatch==8.4",
     "dill==0.3.4",
     "pytz",
     "python-snappy==0.6.1",
 ]
 
-TEST_REQUIREMENTS = ["requests-mock~=1.9.3", "pytest-mock~=3.6.1", "pytest~=6.1", "pandas==2.0.3", "docker", "moto"]
+TEST_REQUIREMENTS = ["requests-mock~=1.9.3", "pytest-mock~=3.6.1", "pytest~=6.1", "pandas==2.0.3", "docker", "moto==4.2.14"]
```

Comment:
Why pin moto?

Reply:
We're using something that's been deprecated - `mock_sts`.

Reply:
Do we have a migration plan for this? Does it affect production code paths or only tests?

Reply:
It just affects tests. There is no migration plan yet but if I can actually get a green build I will make a ticket.